rag_app_sample_code/A_POC_app/pdf_uc_volume/04_create_evaluation_set.py

```diff
@@ -108,6 +108,20 @@
     F.col("is_correct"),
 ))
 
+# COMMAND ----------
+# MAGIC %md
+# MAGIC ## Ingest chats into the Review App to be reviewed 
+
+# COMMAND ----------
+
+from databricks.agents import enable_trace_reviews
+
+request_id_list = request_log_df.select(F.col("request_id")).rdd.flatMap(lambda x: x).collect()
```

Comment:
QQ, why do we need the `.rdd.flatMap` call? It might be cleaner to e.g. use `.toPandas` instead

Reply:
Good point. Simplified below.